modules/bastion/main.tf

```diff
@@ -77,15 +77,15 @@ resource "azurerm_subnet_network_security_group_association" "graphdb_bastion" {
 }
 
 resource "azurerm_public_ip" "graphdb_bastion" {
-  name                = "${var.resource_name_prefix}_bastion_publicIP"
+  name                = "pip-${var.resource_name_prefix}-bastion"
   location            = var.location
   resource_group_name = var.resource_group_name
   allocation_method   = "Static"
   sku                 = "Standard"
 }
 
 resource "azurerm_bastion_host" "graphdb" {
-  name                = "${var.resource_name_prefix}_bastion"
+  name                = "bas-${var.resource_name_prefix}"
```

Comment:
why not "bh" (bastion host) ? "bas" could be misleading

Reply:
https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations